src/functionBased/Components/InputTodo.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+
+const InputTodo = (props) => {
```

Comment:
Please try not to disable linters, it would be better if you fix linter errors instead